nx.json

```diff
@@ -112,7 +112,7 @@
     "@fluentui/react-storybook": { "implicitDependencies": [] },
     "@fluentui/react-tabs": { "implicitDependencies": [] },
     "@fluentui/react-tabster": { "implicitDependencies": [] },
-    "@fluentui/react-text": { "implicitDependencies": [] },
+    "@fluentui/react-text": { "tags": ["vNext"], "implicitDependencies": [] },
```

Comment:
there is new functionality/fix in migration generator - consider to rebase and re-run the migration on your component